internal/compiler/output_columns.go

```diff
@@ -159,14 +161,18 @@ func outputColumns(qc *QueryCatalog, node ast.Node) ([]*Column, error) {
 						return nil, err
 					}
 					for _, c := range columns {
-						found = true
-						c.NotNull = true
-						c.skipTableRequiredCheck = true
-						cols = append(cols, c)
+						if firstColumn == nil {
+							firstColumn = c
+						}
+						shouldNotBeNull = shouldNotBeNull || c.NotNull
```

Comment:
Question: What are the differences between `n.Args.Items` and their inner `columns`?